x-pack/plugins/siem/server/lib/timeline/routes/create_timelines_route.ts

```diff
@@ -46,37 +40,43 @@ export const createTimelinesRoute = (
         const frameworkRequest = await buildFrameworkRequest(context, security, request);
 
         const { timelineId, timeline, version } = request.body;
-        const { templateTimelineId, timelineType } = timeline;
-        const isHandlingTemplateTimeline = timelineType === TimelineType.template;
+        const { templateTimelineId, templateTimelineVersion, timelineType } = timeline;
 
-        const existTimeline =
-          timelineId != null ? await getTimeline(frameworkRequest, timelineId) : null;
-        const existTemplateTimeline =
-          templateTimelineId != null
-            ? await getTemplateTimeline(frameworkRequest, templateTimelineId)
-            : null;
+        const timelineStatus = new TimelinesStatus({
+          timelineType: timelineType ?? TimelineType.default,
+          timelineInput: {
+            id: timelineId ?? null,
```

Comment:
do we need all `null` here? maybe instead we could move that into `TimelineStatus` constructor, so we don't need to duplicate `null` everywhere we will use `TimelineStatus`